webapp/views.py

```diff
@@ -468,12 +468,26 @@ def engage_page(language, page):
         if not metadata:
             flask.abort(404)
         else:
+            related_pages_metadata = []
+            if "related_urls" in metadata:
+                if metadata["related_urls"].strip() != "":
+                    related_urls = metadata["related_urls"].split(",")
+                    # Only show maximum of 3 related pages
+                    for i in range(3):
```

Comment:
I'm setting a strict limit here since we only allow max 3 related posts to be shown and I can't set the limit on the discourse template itself. Let me know if there's a better suggestion to do this.

Reply:
Left a small suggestion. This is a really small list so I don't think there's much difference in performance, just a tiny bit dryer